### PR TITLE
feat: use crowdin branches

### LINKED ISSUE #1674: Splitting into version branches
Currently, translation contents are stored in versions folders. But doing this occurs some confusing on Crowdin because there are many contents like these:

- /current/en-US (should be translated)
- /en-US (not used)
- /(major)-x-y/en-US (not used except latest version)

According [Crowdin docs](https://support.crowdin.com/versions-management/), I think it needs to split them into their version branches:

- [ ] 9-x-y
- [ ] 10-x-y
- [ ] 11-x-y (current)
- [ ] 12-x-y

### PR DESCRIPTION
**Pull request by vhashimotoo**
Fixes #1674

### CODE REVIEW

**Review by molant — COMMENTED** [review]
Thanks @vhashimotoo!
I was hoping to switch to the GitHub integration instead of GitHub actions to reduce some of the code, but I'm having some problems with my tests so probably better stick to what we know is working 😊

Are you planning on moving the translations of each matching branch to `/content/`?

**Comment by vhashimotoo** [comment]
> Are you planning on moving the translations of each matching branch to `/content/`?

I expect Crowdin to automatically do this

File: .github/workflows/download-content.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  content-11-x-y:
+    name: Fetch latest translated content (11-x-y)
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@master
+        with:
+          repository: vhashimotoo/i18n-content
+          ref: content/11-x-y
+      - name: Download content from Crowdin
+        uses: crowdin/github-actions@1.0.9
```
[the remark this answers is not in the document]
Comment (vhashimotoo) [reply .github/workflows/download-content.yml:39]: Nope, haven't looked, thanks, will update

**Comment by molant** [comment]
> I expect Crowdin to automatically do this

I didn't get the feeling they'd modify paths of the content when I talked to them about versioning a couple months back 🤔. Did you find something in the documentation that made you think that? I can probably pinged them to clarify if needed.

**Pull request closed without merging**